packages/schematics/schematics/blank/project-files/package.json.template

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@angular-devkit/core": "^<%= coreVersion %>",
     "@angular-devkit/schematics": "^<%= schematicsVersion %>",
-    "typescript": "~3.9.7"
+    "typescript": "~4.0.2-insiders.20200818"
```

Comment:
Is this version intentional? “schematics/schematic” contains this version as well.

Reply:
Oops missed that.